packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+
+Adds a Group target for a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a User
+
+When you add the first Group target, you reduce the scope of the role assignment. The Role no longer applies to all targets, but applies only to the specified target.
+
+###### Request parameters
+
+
+| Parameter   | Description                                     | Param Type   | DataType   | Required |
+| :---------- | :---------------------------------------------- | :----------- | :--------- | :------- |
+| groupId     | `id` of Group target to scope role assignment   | URL          | String     | TRUE     |
+| roleId      | `id` of a Role                                  | URL          | String     | TRUE     |
+| userId      | `id` of a User                                  | URL          | String     | TRUE     |
+
+###### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/groups/00garkxjAHDYPFcsP0g4"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Add Group target to group administrator role given to a group
+
+<ApiOperation method="put" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/groups/${targetGroupId}" />
+
+Adds a Group target for a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a Group
+
+When you add the first Group target, you reduce the scope of the role assignment. The Role no longer applies to all targets, but applies only to the specified target.
+
+###### Request parameters
+
+
+| Parameter            | Description                                     | Param Type   | DataType   | Required |
+| :------------------- | :---------------------------------------------- | :----------- | :--------- | :------- |
+| groupId              | `id` of an admin Group                          | URL          | String     | TRUE     |
+| roleId               | `id` of a Role                                  | URL          | String     | TRUE     |
+| targetGroupId        | `id` of Group target to scope role assignment   | URL          | String     | TRUE     |
+
+###### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/JBCUYUC7IRCVGS27IFCE2SKO/targets/groups/00gsrhsUaRoUib0XQ0g3"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Remove a group target from a group administrator role
+
+##### Remove a group target from a group administrator role given to a user
```

Comment:
group target from a group administrator role given to a user > Group target from a Group administrator Role given to a User